org-formation/600-access/organization-tasks.yaml

```diff
@@ -6,11 +6,76 @@ NlpSandboxServiceAccount:
   StackName: NlpSandboxServiceAccount
   MaxConcurrentStacks: 10
   Parameters:
-    ManagedPolicyArns:
-      - arn:aws:iam::aws:policy/AmazonEC2FullAccess
-      - arn:aws:iam::aws:policy/AWSCloudFormationFullAccess
-      - arn:aws:iam::aws:policy/AmazonS3FullAccess
+    PolicyDocument: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "ec2",
+                "Effect": "Allow",
+                "Action": [
+                    "ec2:AuthorizeSecurityGroupEgress",
+                    "ec2:AuthorizeSecurityGroupIngress",
+                    "ec2:CreateTags",
+                    "ec2:CreateSecurityGroup",
+                    "ec2:DescribeInstances",
+                    "ec2:DescribeSecurityGroups",
+                    "ec2:DescribeSubnets",
+                    "ec2:DeleteSecurityGroup",
+                    "ec2:DeleteTags",
+                    "ec2:RevokeSecurityGroupEgress",
+                    "ec2:RunInstances",
+                    "ec2:TerminateInstances",
+                    "elasticloadbalancing:AddTags",
+                    "elasticloadbalancing:DescribeTargetGroups",
+                    "elasticloadbalancing:DescribeTargetHealth",
+                    "elasticloadbalancing:DescribeLoadBalancers",
+                    "elasticloadbalancing:DescribeListeners",
+                    "elasticloadbalancing:DeleteTargetGroup",
+                    "elasticloadbalancing:DeleteLoadBalancer",
+                    "elasticloadbalancing:RemoveTags",
+                    "elasticloadbalancing:CreateListener",
+                    "elasticloadbalancing:CreateTargetGroup",
+                    "elasticloadbalancing:CreateLoadBalancer",
+                    "elasticloadbalancing:RegisterTargets"
+                ],
+                "Resource": "*"
```

Comment:
I can't seem to figure out the condition, I tried doing something like this:

```
"Condition": {
    "StringLike": {
        "aws:ResourceTag/name": "nlpsandbox-*"
    }
}
```

Reply:
What we did with regular EC2 is to add a deny rule and a conditional exception to that rule: https://github.com/Sage-Bionetworks/aws-infra/blob/9ce5fae0fdd223eb25fddef51c7291c97c268d7c/templates/essentials.yaml#L410

I'm not sure how that model translates to this service entity...

Reply:
@tschaffter , here is the condition that needs to be worked on